src/libraries/System.ComponentModel.EventBasedAsync/src/System/ComponentModel/AsyncOperation.cs

```diff
@@ -9,14 +9,14 @@ namespace System.ComponentModel
     public sealed class AsyncOperation
     {
         private readonly SynchronizationContext _syncContext;
-        private readonly object _userSuppliedState;
+        private readonly object? _userSuppliedState;
```

Comment:
Do we still need this private field? Is seems to only be used by `UserSuppliedState` property.

Reply:
It's set in the ctor and returned by the property.  Do you mean just making it an auto-prop?  We can do that.

Reply:
Yeah that's what I meant. I saw you did it in other classes as part of this change, so that's why I suggested it.